airflow/cli/commands/fastapi_api_command.py

```diff
@@ -124,6 +125,10 @@ def fastapi_api(args):
             "python:airflow.api_fastapi.gunicorn_config",
         ]
 
+        ssl_cert, ssl_key = _get_ssl_cert_and_key_filepaths(args)
+        if ssl_cert and ssl_key:
+            run_args += ["--certfile", ssl_cert, "--keyfile", ssl_key]
```

Comment:
Could you please add a test for adding these flags to the command?

Reply:
Have added to `test_cli_fastapi_api_args`.